packages/api/lambdas/reports/granule-inventory-report.js

```diff
@@ -38,6 +30,24 @@ async function createGranuleInventoryReport(recReportParams) {
     'updatedAt',
     'published',
   ];
+
+  const { reportKey, systemBucket, ...params } = recReportParams;
+  const searchParams = {};
+  fields.forEach((field) => {
+    let searchField = field;
+    if (field === 'granuleUr') searchField = 'granuleId';
```

Comment:
Are they not named the same thing?

Reply:
The csv column is called `granuleUr`, while the API accepts `granuleId`. Didn't want to change the output, but wanted to use the fields as an easy way of getting relevant search params, so just had it replace the name here.